tests/integration/server/api/roles_api_test.py

```diff
@@ -268,6 +273,19 @@ def test_add_role_owner():
         proposal_response = get_proposal_with_retry(session, result["proposal_id"])
         proposal = assert_api_success(proposal_response)
         assert proposal["data"]["assigned_approver"][0] == user1_id
+        # Logging in as role owner
+        credentials_payload = {
+            "id": user1_payload["username"],
+            "password": user1_payload["password"],
+        }
+        log_in(session, credentials_payload)
+        # Approve proposal as role owner
+        approval_response = approve_proposal(session, result["proposal_id"])
+        LOGGER.info(approval_response)
```

Comment:
I think we try to keep logging statements out of tests.

Reply:
Yep, when I moved on to new tests to explicitly check the outbound queue, I forgot to go back and clean out the logging from these.